shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/widget/presentation/StudyPlanWidgetFeature.kt

```diff
@@ -6,9 +6,13 @@ import org.hyperskill.app.learning_activities.domain.model.LearningActivityType
 import org.hyperskill.app.study_plan.domain.model.StudyPlan
 import org.hyperskill.app.study_plan.domain.model.StudyPlanSection
 import org.hyperskill.app.track.domain.model.Track
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
 
 object StudyPlanWidgetFeature {
 
+    val StudyPlanFetchInterval: Duration = 1.seconds
```

Comment:
Probably could be `internal` or `private`

Reply:
I think we follow this code style for constants `StudyPlanFetchInterval` -> `STUDY_PLAN_FETCH_INTERVAL` or an all-caps version of the snake case is Java style and in Kotlin another code style guide for naming constants?

Reply:
Yes, you are right. Constants-like properties should be named in a screaming-snake-case way.
https://kotlinlang.org/docs/coding-conventions.html#property-names